db/migrate/20221206220527_create_notifications.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNotifications < ActiveRecord::Migration[7.0]
+  def change
+    create_table :notifications do |t|
+      t.belongs_to :user, null: false, foreign_key: true
+      t.string :notification_type
```

Comment:
Can we restrict here which kind of notification_types are accepted?

Reply:
Yes, notification type is an enum.